examples/wagmi/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.3",
   "private": true,
   "dependencies": {
-    "@safe-global/safe-apps-provider": "^0.15.2",
+    "@safe-global/safe-apps-provider": "^0.16.0",
```

Comment:
the changeset library should handle all the version changes afterwards, no need to update them manually